Makefile

```diff
@@ -30,5 +30,10 @@ setup:  ## locally setup app and users. presumes env vars are set
 	echo "Creating directories and configs"
 	./scripts/setup_system.sh
 
+.PHONY: setup-mta-sts
+setup-mta-sts:  ## setup MTA-STS and TLS-RPT. usage: make setup-mta-sts DOMAIN=example.com MX=mail.example.com EMAIL=admin@example.com
+	@if [ -z "$(DOMAIN)" ]; then echo "Error: DOMAIN is required. Usage: make setup-mta-sts DOMAIN=example.com MX=mail.example.com EMAIL=admin@example.com"; exit 1; fi
+	@if [ -z "$(MX)" ]; then echo "Error: MX is required. Usage: make setup-mta-sts DOMAIN=example.com MX=mail.example.com EMAIL=admin@example.com"; exit 1; fi
+	./scripts/setup_mta_sts.sh -d $(DOMAIN) -m "$(MX)" $(if $(EMAIL),-e $(EMAIL),)
```

Comment:
If you removed this script, you should remove this section in the makefile as well, right?